packages/widget/src/types/events.ts

```diff
@@ -5,13 +5,22 @@ export enum WidgetEvent {
   RouteExecutionUpdated = 'routeExecutionUpdated',
   RouteExecutionCompleted = 'routeExecutionCompleted',
   RouteExecutionFailed = 'routeExecutionFailed',
+  RouteHighValueLoss = 'highValueLoss',
```

Comment:
Let's name it the same everywhere, so it should be 

`RouteHighValueLoss = 'routeHighValueLoss',`